airbyte-integrations/connectors/source-mssql/src/main/resources/spec.json

```diff
@@ -31,6 +31,59 @@
         "description": "Password associated with the username.",
         "type": "string",
         "airbyte_secret": true
+      },
+      "ssl_method": {
+        "title": "SSL Method",
+        "type": "object",
+        "description": "Encryption method to use when communicating with the database",
+        "order": 6,
+        "oneOf": [
+          {
+            "title": "Unencrypted",
+            "additionalProperties": false,
+            "description": "Data transfer will not be encrypted.",
+            "required": ["ssl_method"],
+            "properties": {
+              "ssl_method": {
+                "type": "string",
+                "enum": ["unencrypted"],
+                "default": "unencrypted"
+              }
+            }
+          },
+          {
+            "title": "Encrypted (trust server certificate)",
+            "additionalProperties": false,
+            "description": "Use the cert provided by the server without verification.  (For testing purposes only!)",
```

Comment:
@heade Not sure weather this option we need to provide to end users. 
@sherifnada Maybe we can leave here a todo, to move activation of this option to some dev specific profile?

Reply:
seems useful to leave it here in case a user is testing locally. Are you worried it will cause user error by using it unintentionally?